src/main/scala/com/gu/elasticsearchmonitor/Lambda.scala

```diff
@@ -22,31 +18,50 @@ object Env {
   def apply(): Env = Env(
     Option(System.getenv("App")).getOrElse("DEV"),
     Option(System.getenv("Stack")).getOrElse("DEV"),
-    Option(System.getenv("Stage")).getOrElse("DEV")
-  )
+    Option(System.getenv("Stage")).getOrElse("DEV"))
 }
 
 object Lambda {
 
   val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
-  /*
-   * This is your lambda entry point
-   */
   def handler(lambdaInput: LambdaInput, context: Context): Unit = {
```

Comment:
I still think it's useful to note that this is the entry point. It may also be worth explicitly mentioning that it's referenced in Cloudformation (it's useful to be reminded of this if you rename the file or method)